### ISSUE TITLE
Colors in grep output

### ISSUE THREAD

**Issue by p-kolacz** [vifm/vifm#784]
It would be useful to see `:grep` output with colors. I tried to add `--color=always` to `grepprg` but color escape sequences are not evaluated in output window.

**Comment by xaizek** [comment]
Yes, menus don't process escape sequences. View mode does and maybe that code can be used in menus too, but not sure how much effort it would take to integrate it and keep everything working properly (there are differences between preview and menus).
